app/views/metadata_presenter/page/confirmation.html.erb

```diff
@@ -7,6 +7,16 @@
       <%= @page.heading %>
     </h1>
 
+    <% if reference_number_enabled? %>
+      <div class="govuk-panel__body">Your reference number</div>
+      <div class="govuk-panel__body">
+        <strong>
+          <%= show_reference_number %>
+        </strong>
+      </div>
+      <br>
+    <% end %>
+
```

Comment:
We really only want one `govuk-panel__body`class.  So probably a bit more like this:

```
<div class="govuk-panel__body">
  <% if reference_number_enabled? %>
    <div>Your reference number</div>
    <div>
      <strong>
        <%= show_reference_number %>
      </strong>
    </div>
    <br>
  <% end %>

  <%# Do not replace with app/views/metadata_presenter/attribute _lede.html.erb because that is different %>
  <% if @page.lede %>
    <div class="fb-editable"
           data-fb-content-id="page[lede]"
           data-fb-content-type="element"
           data-fb-default-text="<%= default_text('lede') %>">
      <%= @page.lede %>
    </div>
  <%- end %>
</div>
```

I presume the `<br>` is for spacing?  You could try and replace the reference number related `<div>`s with `<p>` tags, which would be more semantic, but I'm not certain how that will render. Happy to pair to refine it a bit if needed...

Reply:
All done!

Reply:
Yep, the `<br>` is for spacing - will give the `<p>` a try

Reply:
Thanks Chris - all done